### ISSUE TITLE
v3 0.12.1

### ISSUE THREAD

**Issue by sm6usu** [ExpertSDR3/ExpertSDR3-BUG-TRACKER#324]
![ExpertSDR3 SunSDR2DX EED06202100558 2022-03-14 21_40_27](https://user-images.githubusercontent.com/101463570/158258434-b8ba9b46-3aed-46cc-8d25-e9660e8e32cf.png)
Happens several times every day
73 de sm6usu.

**Comment by danweecc** [comment]
Same issue as reported here:-
https://github.com/ExpertSDR3/ExpertSDR3-BUG-TRACKER/issues/275

**Comment by sm6usu** [comment]
Ok my tranceiver is not remote but I have Log4om running  to WiFi for cluster and spots. Maybe try to run whithout to see if it helps . Laptop is fairly new supported wifi 6 and started this behaivor when I started up using 0.12.1 Never before that. Thanks for answer still learning. Hi
73 de sm6usu

**Comment by danweecc** [comment]
It does not have to be remote. Latency can also be elevated for other reasons, such as using a Wi-Fi connection instead of a LAN connection. If your laptop where you're running EESDR3 is connected using Wi-Fi, there is a good chance this is causing elevated latencies and EESDR3 0.12.x appears to be quite sensitive to this and tends to drop the connection as a result. It did not happen prior to 0.12.0 nor with EESDR2.

**Comment by sm6usu** [comment]
Ok thank you I will look in to my equipment here in my house.
sm6usu Svante


Den tis 15 mars 2022 07:15danweecc ***@***.***> skrev:

**Comment by sm6usu** [comment]
New router today wifi 6 on computer and router, new ehternet cable cat 6. still same problem. Tranceiver stop just now  when sitting here and writing this report for the second  time in 10 minits. I talk to a guy yesterday and he had adapt a second network card and had no problem in 2 days. But I dont know. I had no problem before 12.1 Maybe go back to 12.0
73 de sm6usu

**Comment by danweecc** [comment]
I think any WiFi will have added latencies unfortunately. If EE programmers can loosen the timing requirements a bit, it would be much better.

**Comment by sm6usu** [comment]
Reinstalled SW yesterday aftenoon. also installed radiosync. Not sure what radiosync does in this case but so far no problem in 5 hours. I will come back here and report.
Thank you for info.
sm6usu Svante

**Comment by sm6usu** [comment]
It disconnected a couple of times today. So now I have bought a cable and a netcard and made my laptop to an Hybrid cable netcard and wifi card and we will se if it helps 1 hour no problems :-)
sm6usu Svante

**Comment by N8SDR007** [comment]
A suggestion in the network metrics of both the NETWORK card and the Network WIFI adapter change the ethernet to a LOWER metric number than the Wi-Fi- Example make NIC card 1 and make Wi-Fi priority say 100, this will give higher attention to the NIC card and not the Wi-Fi adapter overall:  I mentioned this with audio dropouts as well, but it applies to disconnects also  https://eesdr.com/en/forum-en/other-questions/9085-possible-help-for-those-experiencing-audio-drop-out-on-windows-10-systems#18348

**Comment by sm6usu** [comment]
Thank you Rick for advice. I did that for about 4 hours ago now with a followed restart. Computer and Tranceiver has now running since Saterday 2 o clock on the day. Still running but I had 1 disconect yesterday and lower the voltage to 13.2 but I not sure that is my problem. (From earlier manual) · "It is forbidden to use the power supply with the voltage higher than +16 V". So what is the max power you can run the sunsdrdx 2 with V3 0.12.1 ? I asking also because my boat where I have suncells pover regulated 12 system somtimes can put in 14.9 volt to the battery bank. Is it ok to use sunsdrdx in my boat?  73 de sm6usu Svante

**Comment by sm6usu** [comment]
Hallo again I am done with my testing. This last test was for the power. I use according to my powersupply 13.8 Volt and according to my sunsdrdx 2 12.8 Volt Tranceiver worked 16.5 hours before it disconnect. Start again and then it works again.
I can also see that the 12,8 volt on screen drops 0.3 to 1.0 volt (ssb) as soon I transmit using 100 watt on my Drive. My power supply does not drop anything. Waiting for a new update now. First time I see disconnect was after upgraded to 78.8 But I also didn,t use my tranceiver so long period that I have done now on my testing week. But of course it schould work 24/7
73 de sm6usu Svante

**Comment by danweecc** [comment]
Are you using the supplied power cable? The voltage drop sounds a little large and I suspect either your power supply cable resistance is high, or have poor powerpole connections.

**Comment by N8SDR007** [comment]
You can also shorten the cable of it much longer then needed that will drop resistance

⁣Get BlueMail for Android ​

On Mar 23, 2022, 6:47 AM, at 6:47 AM, danweecc ***@***.***> wrote:
>Are you using the supplied power cable? The voltage drop sounds a
>little large and I suspect either your power supply cable resistance is
>high, or have poor powerpole connections.
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/ExpertSDR3/ExpertSDR3-BUG-TRACKER/issues/324#issuecomment-1076221825
>You are receiving this because you commented.
>
>Message ID:
***@***.***>

**Comment by sm6usu** [comment]
Yes using original it is long but sometimes that is good ( if using in my
boat) So I better try to get an extra shorter one.
Thank you
73 de sm6usu Svante

Den ons 23 mars 2022 kl 11:47 skrev danweecc ***@***.***>:

**Comment by sm6usu** [comment]
Yes Rick I will try to get an extra cable. It is sometimes good to have 2
different cables.
73 de sm6usu Svante

Den ons 23 mars 2022 kl 12:27 skrev Rick N8SDR ***@***.***>:

> You can also shorten the cable of it much longer then needed that will
> drop resistance
>
> ⁣Get BlueMail for Android ​
>

**Comment by sm6usu** [comment]
Hallo everyone. I have now No issues with Disconnecting the last 5 days. The problem was the Bandscope/Navidia settings. I did a zoom on bandscope 312 kHz no problem, untill I hit the ptt .The zoom hangs up with big delays. Tried 156 kHz no problem could zoom fast and no problem. But today, I get a couple of times with distortied rx audio in qso,s I just disconnect and connect with "on and of top left" and problem was gone. Now I am trying 78 kHz to see what happens. I know it is a new bandscope on next update so hope that solve the issues. I have also set my Navidia RTX 3070 Gpu card to my priorety nr.1 in my laptop, which have 2 Gpu. According to manual for laptop it schould handle 2 external 4K screens. I have only 1. 27 inch screen with 4 K. So I will now Close this case do to no issues with disconnecting If running Lower bandscop
